community/sway/etc/sway/config.d/99-autostart-applications.conf

```diff
@@ -11,10 +11,11 @@ exec {
     '[ -x "$(command -v flashfocus)" ] && flashfocus'
     '[ -x "$(command -v wlsunset)" ] && /usr/share/sway/scripts/sunset.sh'
     '[ -x "$(command -v autotiling)" ] && autotiling'
+    '[ -x "$(command -v sworkstyle)" ] && sworkstyle'
```

Comment:
```suggestion
    '[ -x "$(command -v sworkstyle)" ] && sworkstyle &> /tmp/sworkstyle.log'
```

Reply:
I also saw it in the official documentation, but I am not sure about the interaction between sway and child processes of sway. What I mean is: I can't recall if sway kills only "exec_always" processes reload or "exec" are closing too. This is kind of important, because with the disown (&) behavior in the command, it cannot be killed. Recently sworkstyle added a .lock file to prevent multiple instances from running at the same time (as it loads CPU to 100%), so it isn't a concern anyway by now, but I'm interested in what's the most "correct" way to do this

Reply:
but if that is true, would it be safe to just not disown it?

Reply:
regarding reload: 99.9% sure it doesn't kill things from "exec". otherwise for example autotiling would be dead after reload, right?

Reply:
Yeah, makes sense, for autotiling it to work then it would have to be exec_always. Nevermind then, it's alright, after all this was a recommendation from the developer of sworkstyle, it's just that I've faced issues with disowning kanshi, for example, but that's an entirely different story, not applicable here